src/test/java/AlenaDubrouskayaTest.java

```diff
@@ -0,0 +1,26 @@
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import runner.BaseTest;
+
+public class AlenaDubrouskayaTest extends BaseTest {
+
+    @Test
+    public void TitleTest() {
```

Comment:
```suggestion
    public void titleTest() {
```
Methods should start with lower case.